Documentation/policy/language.rst

```diff
@@ -301,6 +301,11 @@ have ``head:none`` set as the label.
 
         .. literalinclude:: ../../examples/policies/l3/service/service-labels.json
 
+Limitations
+~~~~~~~~~~~
+
+``toServices`` statements cannot be combined with ``ToPorts`` statements in the
```

Comment:
Should this be `toPorts` instead of `ToPorts`? I think the former is used in CNPs written by the user while the latter is the name in the agent-internal representation of a CNP? Or am I mixing something up?

```suggestion
``toServices`` statements cannot be combined with ``toPorts`` statements in the
```

Reply:
I think you're right, I'll fix this up  :+1: